src/main/groovy/com/f2prateek/javafmt/AndroidJavaFmtExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package com.f2prateek.javafmt;
+
+class AndroidJavaFmtExtension {
+  boolean aospStyle = false
```

Comment:
I was mostly thinking about it in regards to making the version of java-format configurable. Probably not necessary since customizability is a non-goal of that project. I'll change it to an enum.